app/domain/audits/plan.rb

```diff
@@ -0,0 +1,46 @@
+module Audits
+  class Plan
+    def self.document_type_ids
+      values.values
+    end
+
+    def self.document_types
+      values
+    end
+
+    def self.values
```

Comment:
It looks like the ordering of the values is significant? If so, I don't think that is a domain concern, but something we should do in the dropdown helper:

```rb
Audits::Plan.document_types.sort_by { |key, value| key.split(/\s>\s/) }
```

Reply:
Addressed in 75f0ed0